sesam.py

```diff
@@ -2212,8 +2257,46 @@ def format(self, record):
     parser.add_argument('-force', dest='force', required=False, action='store_true',
                         help="force the command to run (only for 'upload' and 'download' commands) for non-dev "
                              "subscriptions")
+
+    parser.add_argument("--system-placeholder", metavar="<string>",
+                        default="xxxxxx", type=str, help="Name of the system _id placeholder")
+
+    parser.add_argument("-d", dest="connector_dir", metavar="<string>",
+                        default=".", type=str, help="Connector folder to work with")
+
+    parser.add_argument("-e", dest="expanded_dir", metavar="<string>",
+                        default=".expanded", type=str, help="Directory to expand the config into")
+
+    parser.add_argument("--client_id", metavar="<string>",
+                        type=str, help="oauth client id")
+
+    parser.add_argument("--client_secret", metavar="<string>",
+                        type=str, help="oauth client secret")
+
+    parser.add_argument("--service_url", metavar="<string>",
+                        type=str, help="url to service api (include /api)")
+
+    parser.add_argument("--service_jwt", metavar="<string>",
+                        type=str, help="jwt token to the service api")
+
+    parser.add_argument("--consumer_token", metavar="<string>",
+                        type=str, help="consumer token")
+
+    parser.add_argument("--employee_token", metavar="<string>",
+                        type=str, help="employee token")
+
+    parser.add_argument("--base_url", metavar="<string>",
+                        type=str, default="https://api.tripletex.io", help="override to use prod env")
+
+    parser.add_argument("--days", metavar="<string>",
+                        type=int, default=10, help="number of days until the token should expire")
+
+    parser.add_argument("--login_service", metavar="<string>",
+                        type=str, default="oauth", help="login service to use", choices=["oauth", "tripletex"])
+
     try:
         args = parser.parse_args()
+        args.use_connector = os.path.exists("manifest.json") or os.path.exists(os.path.join(args.connector_dir, "manifest.json"))
```

Comment:
I would change the name to `is_connector` as it's about whether "are we working on a connector repository". :)

Reply:
It makes sense. we can use os.path.exists(os.path.join(args.connector_dir, "manifest.json")) and this will cover both cases. Just in case of wrongly passing connector_dir as arg while the workdir is the connector, this will be false while the combined condition will still pass. Maybe that case is negligible and we can just use the simpler condition since it is incorrect user input.

Btw, it's good that you gave a comment here. I think I should have used os.path.isfile instead. That makes more sense. I will change that too.

Reply:
I see what you mean with the edge case - up to you, it's not bad with the `or` condition either. It's really a question of should a "wrong" user input still work or not. You could argue that we should let the user know that their input is wrong (i.e. both setting the arg to something and having the manifest in the workdir).

Reply:
For now I have removed the other condition. Then we can discuss that. Maybe it's good to log the is_connector so the user will get informed about the status. We can have it as a small change in another PR.